### DISCUSSION TITLE
Merging can be performed automatically with 0 approving reviews

### DISCUSSION THREAD

**Discussion by raphlcx** [community/community#15890] [Pull Requests]
Hi,

I'm attempting to merge a pull request into my default branch, named "main".

This screen shows the (one and only) branch protection rule for "main" branch:

<img width="779" alt="Screenshot 2022-05-02 at 01 00 27" src="https://user-images.githubusercontent.com/8281591/166156647-df76e6ce-3fd1-40f0-9309-598ff43d6a82.png">

On the pull request:

<img width="643" alt="Screenshot 2022-05-02 at 01 00 38" src="https://user-images.githubusercontent.com/8281591/166156665-9c472212-ed05-4b05-be1f-510af2794ae6.png">

It is understandable that the PR is not mergeable yet without an approval from Codeowners.

But, even though the "Require approvals" under branch protection is not enabled, it is still weird to see:

```
 Merging can be performed automatically with 0 approving reviews.
```

If it requires 0 approving reviews, it would mean the PR is mergeable now, which it is not.

**Comment by juju4** [comment]
I got same issue.
Just ensure that in the protected branch, there is no setting requiring approval.
Not just "Require approvals" but also "Require approval of the most recent reviewable push"

**Reply by TWiStErRob** [reply]
Yes, the minimal repro is
![image](https://user-images.githubusercontent.com/2906988/221505525-3051bedd-40df-4c55-b6da-0221256c6866.png)

but GitHub is really good at not letting us do silly things. For example you can only select number of approvals, if they're enabled:
![image](https://user-images.githubusercontent.com/2906988/221505713-656d71d2-c65b-425a-b207-d761089a4d6f.png)

I smell a bit of UX improvements here.

**Comment by ProgrammerDurgesh** [comment]
![image](https://github.com/community/community/assets/50780492/78d9c42e-2356-41fb-b97f-ee40b739316b)

**Reply by TWiStErRob** [reply]
What are you trying to say?

**Comment by ashishjullia** [comment]
Hi, I also got a similar issue here: https://github.com/orgs/community/discussions/67622

Am I missing something @raphlcx @TWiStErRob

**Comment by jm-alan** [comment]
FYI the ruleset UI has been updated since [this related question's answer](https://github.com/orgs/community/discussions/74977) and it's no longer possible to decouple required approvals from requiring a PR before merging; the checkbox to disable "require approvals" is entirely gone, and setting required approvals to 0 still produces this same problem over a year later.
